Library/Formula/rock.rb

```diff
@@ -3,18 +3,10 @@
 class Rock < Formula
   desc "ooc compiler written in ooc"
   homepage 'http://ooc-lang.org'
-  url 'https://github.com/nddrylliog/rock/archive/v0.9.9.tar.gz'
-  sha1 '11d4a46320e2b538989354505e0a6ac4311b049f'
+  url 'https://github.com/fasterthanlime/rock/archive/v0.9.10.tar.gz'
+  sha1 'c3bc1847d7e1753c66d04a2b333cbb405053d46f'
 
-  head 'https://github.com/nddrylliog/rock.git'
-
-  bottle do
```

Comment:
Leave this block intact please. We'll update it with new bottles automatically when we pull.